firestore/extend-with-functions/functions/index.js

```diff
@@ -151,12 +155,12 @@ function multiWildcard() {
   exports.useMultipleWildcards = functions.firestore
       .document('users/{userId}/{messageCollectionId}/{messageId}')
       .onWrite((change, context) => {
-        // If we set `/users/marie/incoming_messages/134` to {body: "Hello"} then
-        // context.params.userId == "marie";
-        // context.params.messageCollectionId == "incoming_messages";
-        // context.params.messageId == "134";
-        // ... and ...
-        // change.after.data() == {body: "Hello"}
+      // If we set `/users/marie/incoming_messages/134` to {body: "Hello"} then
+      // context.params.userId == "marie";
+      // context.params.messageCollectionId == "incoming_messages";
+      // context.params.messageId == "134";
+      // ... and ...
+      // change.after.data() == {body: "Hello"}
```

Comment:
Just noticed that here you did the opposite of what I suggested in my comment above. Was this done by eslint's autofix? If it was, changing it will be unnecessary and we can leave it as it is.